docker/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+# helper docker-compose file for testing
+
+version: '2.1'
+services:
+  # InfluxDB 2.0 that the initialization is run against
+  influxdb:
+    image: quay.io/influxdb/influxdb:2.0.0-beta
+    ports:
+      - 9999:9999
+    volumes:
+      - influx-cli-configs:/root/.influxdbv2/
+  influx_setup:
```

Comment:
I'd add `scale: 0` so this doesn't run during `docker-compose up`

This would require the user to run `docker-compose run --rm influxdb_setup` to make that work though.

Another approach would be to add `restart: Never` and then the user will only need to run `docker-compose up`, but they may see a failed container after the first run.

Reply:
i added restart: "no"